src/main/java/org/folio/dcb/service/impl/ItemServiceImpl.java

```diff
@@ -55,10 +57,10 @@ public String fetchItemMaterialTypeNameByMaterialTypeId(String materialTypeId) {
   @Override
   public String fetchItemLoanTypeIdByLoanTypeName(String loanTypeName) {
```

Comment:
@DmytroBykov1 , Looks like this method is not being used. Do we need this method ?

Reply:
@Vignesh-kalyanasundaram, I removed it